spec/system/dashboard_system_spec.rb

```diff
@@ -0,0 +1,194 @@
+require "timecop"
```

Comment:
I don't *think* this is necessary because it's in our `Gemfile` -- does the test still pass if you remove it?

Reply:
Good question!